src/main/java/Echo.java

```diff
@@ -0,0 +1,89 @@
+import echo.*;
+
+import java.io.FileNotFoundException;
+
+import java.io.IOException;
+
+public class Echo {
+
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+    public static final String DOCS_TASKS_TXT = "docs/tasks.txt";
```

Comment:
I'm not sure if using relative path will work will when packaging the entire app into a jar and run by different users